app/views/layouts/_sidebar.html.erb

```diff
@@ -40,7 +40,7 @@
       </li>
 
       <li class="sidebar-item">
-        <a href="#" class="sidebar-link">
+        <a href="<%= subjects_path %>" class="sidebar-link">
```

Comment:
Would be better to use `link_to` here

Reply:
Ça complique l'affichage de l'icone par la suite.

Reply:
Nope, c'est très bien fait plus loindans ton core :
`<%= link_to  subject_path(subject.id), method: :delete, data: { confirm: 'Êtes-vous sûr ?' } do %>
   <i class="align-middle" data-feather="trash"></i>
<% end %>`